CMakeLists.txt

```diff
@@ -293,6 +293,7 @@ list(APPEND COMPILER_SRCS "src/target/datatype/myfloat/myfloat.cc")
 file(GLOB RUNTIME_SRCS
   src/runtime/*.cc
   src/runtime/vm/*.cc
+  src/runtime/hexagon/hexagon/hexagon_buffer.cc
```

Comment:
This allows for unit testing of the HexagonBuffer class, but not sure if we can land this change.

Reply:
Moved this to Hexagon.cmake.